src/Shared/FileUtilities.cs

```diff
@@ -243,6 +243,58 @@ internal static string EnsureNoTrailingSlash(string path)
             return path;
         }
 
+        /// <summary>
+        /// Ensures the path is enclosed within single quotes.
+        /// </summary>
+        /// <param name="path"></param>
```

Comment:
Avoid param and returns tags with no content?

Reply:
I'll make a change to add content to the tags.

Reply:
Done